libs/ui-lib/lib/ocm/components/clusterConfiguration/manifestsConfiguration/components/customManifestsValidationSchema.tsx

```diff
@@ -57,19 +54,9 @@ export const getFormViewManifestsValidationSchema = Yup.lazy<ManifestFormData>((
           .concat(getUniqueValidationSchema(getAllManifests)),
         manifestYaml: Yup.string()
           .required('Required')
-          .test('not-big-file', getMaxFileSizeMessage(MAX_FILE_SIZE), (value: string) =>
-            validateFileSize(value, MAX_FILE_SIZE),
-          )
+          .test('not-big-file', getMaxFileSizeMessage(), validateFileSize)
```

Comment:
`getMaxFileSizeMessage` doesn't have to be a function.

Also, I'm getting a type warning in my editor. It dissapears when I move the type parameter from `Yup.array<...>()` to `Yup.object<...>()`.